doc/how_to_build_linux.md

```diff
@@ -97,11 +97,12 @@ OpenToonzPROFILES="$HOME/.config/OpenToonz/stuff/profiles"
 TOONZCACHEROOT="$HOME/.config/OpenToonz/stuff/cache"
 TOONZCONFIG="$HOME/.config/OpenToonz/stuff/config"
 TOONZFXPRESETS="$HOME/.config/OpenToonz/stuff/projects/fxs"
-TOONZLIBRARY="$HOME/.config/OpenToonz/stuff/projects/library"
+TOONZLIBRARY="$HOME/.config/OpenToonz/stuff/library"
 TOONZPROFILES="$HOME/.config/OpenToonz/stuff/profiles"
 TOONZPROJECTS="$HOME/.config/OpenToonz/stuff/projects"
 TOONZROOT="$HOME/.config/OpenToonz/stuff"
 TOONZSTUDIOPALETTE="$HOME/.config/OpenToonz/stuff/studiopalette"
+TOONZFXPRESETS="/home/USERNAME/.config/OpenToonz/stuff/fxs"
 EOF
 ```
```

Comment:
For now this part of instruction is obsolete. CMake generates shell-script named 'opentoonz' in bin directory. This script generates this ini-file automatically if needed and launches OpenToonz_1.1